tests/api/test_axis.py

```diff
@@ -1831,8 +1831,11 @@ class TestAxis360AcsFulfillmentInfo:
     @pytest.fixture
     def mock_request(self):
         # Create a mock request object that we can use in the tests
+        response = MagicMock(return_value="")
+        type(response).headers = PropertyMock(return_value=[])
+        type(response).status = PropertyMock(return_value=200)
```

Comment:
This test needed to be updated, because werkzeug started doing some parameter validation, and the old mock returned a status code of "" (empty string), which caused these tests to fail. Updating the mock to return a valid status code (200) makes the tests pass again.